README.md

```diff
@@ -10,16 +16,16 @@ check if local DB has unsynced changes.
 // Initialisation
 
 var db = new PouchDB('dbname')
-var api = db.hoodieLocalChanges({remote: 'http://example.com/mydb'})
+var api = db.unsyncesLocalDocs({remote: 'http://example.com/mydb'})
```

Comment:
there is no db.unsyncesLocalDocs() that returns an `api` instance, on which you'd call a method any more. There is only a `db.unsyncesLocalDocs` method now, that you can call like this:

```js
db.unsyncedLocalDocs({
  remote: 'http://example.com/mydb',
  // optional
  keys: [id1, id2]
}).then(function (docs) {
  // docs is array of docs
})
```